src/api/routes/company.js

```diff
@@ -57,4 +61,40 @@ module.exports = (app) => {
         }
 
     });
+
+    router.post(
```

Comment:
This should be idempotent (PUT) (if the company is already blocked, do nothing and succeed (no need for canBlock check)

Reply:
I was following the same logic as offers but I can change